From 2927821e14523fa0ee18140aa7ff6e0509b48ab7 Mon Sep 17 00:00:00 2001
From: Darren Hart <dvhart@linux.intel.com>
Date: Tue, 25 Jun 2013 20:08:46 -0600
Subject: [PATCH] PCI: Add CircuitCo vendor ID and subsystem ID

Add CircuitCo's newly created VENDOR ID and their first board subsystem
ID for the MinnowBoard.

[bhelgaas: sort, change DEVICE_ID to SUBSYSTEM_ID]
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>(cherry picked from commit 91bbe923d18cfff4286a84e59b9d5b61389c3027)

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 include/linux/pci_ids.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index c129162..4c7b349 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2476,6 +2476,9 @@
 
 #define PCI_VENDOR_ID_ASMEDIA		0x1b21
 
+#define PCI_VENDOR_ID_CIRCUITCO		0x1cc8
+#define PCI_SUBSYSTEM_ID_CIRCUITCO_MINNOWBOARD	0x0001
+
 #define PCI_VENDOR_ID_TEKRAM		0x1de1
 #define PCI_DEVICE_ID_TEKRAM_DC290	0xdc29
 
-- 
1.8.1.2

